### ISSUE TITLE
get_inst_proposal does not honor default_fs

### ISSUE THREAD

**Issue by e9925248** [yast/yast-storage#178]
src/modules/StorageProposal.rb function get_inst_proposal
```
      root = {
        "mount"       => "/",
        "increasable" => true,
        "fsys"        => PropDefaultFs(),
        "size"        => 0
      }
      opts = GetControlCfg()
```

The call to GetControlCfg needs to be moved before setting up the root structure:
PropDefaultFs() returns proposal_root_fs, which has at that point the default value btrfs - so the proposal is always based on btrfs.
GetControlCfg calls SetProposalDefault, which in turn calls SetProposalRootFs to initialise the root fs type from /etc/sysconfig/storage.  Switching both commands would make sure, that the proposal honors the user settings.

**Comment by ancorgs** [comment]
Please, use openSUSE's bugzilla to report and track errors. http://bugzilla.opensuse.org

**Comment by e9925248** [comment]
https://bugzilla.opensuse.org/show_bug.cgi?id=959435

**Comment by ancorgs** [comment]
@e9925248 Good point. We wrote that before bugzilla/fate reports started to be mandatory. We'll probably need to fix contributing.md. Noted down.

**Comment by ancorgs** [comment]
Closed by #179 (already merged and sent to TW). More info in the corresponding bugzilla entry.
